src/style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: 400;
+  color: #000;
+}
+
+body {
+  margin: 0;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 403px;
+  background-color: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image-block {
+  margin: 29px 19px 45px;
+  height: 134px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product__titles {
+  display: flex;
+  flex-direction: column;
+  margin: 0 14px 15px;
+}
+
+.product__item-title {
+  font-weight: 500;
+  font-size: 12px;
+  padding-bottom: 6px;
+  line-height: 15px;
+}
+
+.product__item-code {
+  display: flex;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+.product__feedback {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 14px 20px;
+  align-items: center;
+}
+
+.product__stars {
+  display: flex;
+}
+
+.product__star-active {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.product__star-not-active {
+  height: 15px;
+  width: 15px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.product__feedback-count {
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.product__price-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 14px 20px;
+}
+
+.product__price-text {
+  font-size: 11px;
```

Comment:
You repeat `font-size: 11px;` three times in your code. Let's move it to the `body`